modules/nf-core/mitohifi/mitohifi/main.nf

```diff
@@ -0,0 +1,63 @@
+process MITOHIFI_MITOHIFI {
+    tag "$meta.id"
+    label 'process_high'
+
+    // Exit if running this module with -profile conda / -profile mamba
+    if (workflow.profile.tokenize(',').intersect(['conda', 'mamba']).size() >= 1) {
+        exit 1, "MitoHiFi module does not support Conda. Please use Docker / Singularity instead."
+    }
+
+    // Docker image available at the biocontainers Dockerhub
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'docker://ghcr.io/marcelauliano/mitohifi:master':
+        'ghcr.io/marcelauliano/mitohifi:master' }"
+
+    input:
+    tuple val(meta), path(reads), path(contigs)
+    path ref_fa
+    path ref_gb
+    val mito_code
+
+    output:
+    tuple val(meta), path("*fasta")                          , emit: fasta
+    tuple val(meta), path("*gb")                             , emit: gb, optional: true
+    tuple val(meta), path("*gff")                            , emit: gff, optional: true
+    tuple val(meta), path("*contigs_stats.tsv")              , emit: stats
```

Comment:
Move this after `fasta`. List the required outputs first and then the optional ones.